index.html

```diff
@@ -58,7 +58,121 @@ <h2 class="present">I’m a software developer</h2>
+            <button type="button" class="btn-card">See Project</button>
+          </article>
+          <article class="card">
+            <img src="#" alt="">
+            <h3>Multi-Post Stories Gain+Glory</h3>
+            <ul class="program-lang">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="btn-card">See Project</button>
+          </article>
+          <article class="card">
+            <img src="#" alt="">
+            <h3>Multi-Post Stories Gain+Glory</h3>
+            <ul class="program-lang">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="btn-card">See Project</button>
+          </article>
+          <article class="card">
+            <img src="#" alt="">
+            <h3>Multi-Post Stories Gain+Glory</h3>
+            <ul class="program-lang">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="btn-card">See Project</button>
+          </article>
+          <article class="card">
+            <img src="#" alt="">
+            <h3>Multi-Post Stories Gain+Glory</h3>
+            <ul class="program-lang">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="btn-card">See Project</button>
+          </article>
+        </div>
+      </section>
+      <section class="about-section">
+        <div class="about">
+          <h2>About me</h2>
+          <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+          <button type="button" class="btn-resume">Get My Resume</button>
+        </div>
+        <div class="card-wrapper">
+          <article class="cards">
+            <img src="./images/icons/icon-languages@2x.svg" class="tools-img" alt="languages">
+            <h3>Languages</h3>
+            <ul class="languages">
+              <li>JavaScript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>CSS</li>
+            </ul>    
+          </article>
+          <article class="cards">
+            <img src="./images/icons/icon-frameworks@2x.svg" class="tools-img" alt="frameworks">
+            <h3>Frameworks</h3>
+            <ul class="languages">
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>Capybara</li>
+              <li>Selenium</li>
+            </ul>      
+          </article>
+          <article  class="cards">
```

Comment:
- [ ] Great job keeping your code clean, but there are some issues we need to fix on these cards, they are not responsive for the 480px screen size, which is supposed to be the starting point for the mobile view, see the difference below 👇 
- Your cards
![image](https://user-images.githubusercontent.com/55465133/129426309-4ec04056-92d9-4805-97e7-46660c38e907.png)

- The original
![image](https://user-images.githubusercontent.com/55465133/129426348-3c2a57fe-8f99-46dd-8cf5-9aa6ae97af61.png)